packages/runtime/container-runtime/src/opLifecycle/remoteMessageProcessor.ts

```diff
@@ -30,14 +36,18 @@ export class RemoteMessageProcessor {
 
 	/**
 	 * Ungroups and Unchunks the runtime ops encapsulated by the single remoteMessage received over the wire
-	 * @param remoteMessage - A message from another client, likely a chunked/grouped op
-	 * @returns the ungrouped, unchunked, unpacked SequencedContainerRuntimeMessage encapsulated in the remote message
+	 * @param remoteMessage - A message from another client, possible virtualized (grouped, compressed, and/or chunked).
+	 * Considered mutable, meaning no other Container or other parallel procedure depends on this object instance.
+	 * @returns the unchunked, decompressed, ungrouped, unpacked SequencedContainerRuntimeMessages encapsulated in the remote message
+	 * Passes through ops that weren't virtualized, e.g. System ops that the ContainerRuntime will ultimately ignore.
```

Comment:
It's not clear to me what "passes through" means. Does it mean "skips", and therefore those skipped ops are not in the return value?

Reply:
Great point, it's not clear given the type.  Updating it.